scripts/utils/train_helper.py

```diff
@@ -248,15 +367,15 @@ def get_adversary(args, total_training_timesteps, train_types, eval_types, curri
                          h_dim=args.ADV_h_dim, 
                          train_types=train_types,
                          has_curriculum= not curriculum.is_random)
-    agent = load_agent(Path(agent_path), args)
```

Comment:
why was this removed? I'm also confused how the call to `generate_TC_for_Adversary()` is working

Reply:
this was a mistake but I don't use this get_adversary function. I use `get_adversary_agent`